### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by JelteF**
This is a draft PR for the 0.2.0 release, all it really contains is a changelog with a curated list of user-facing changes and some doc changes for those features. It also starts to build docker images for PG14.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,48 @@
-# 0.1.0
+# 0.2.0 (2024-11-10)
```
Comment (wuputah) [inline CHANGELOG.md:1]:
```suggestion
# 0.2.0 (2024-12-10)
```

Comment (wuputah) [reply CHANGELOG.md:1]: This looks great! I don't see anything else aside from keeping this up to date with any other changes that get merged.

**Review by Y-- — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JelteF) [reply CHANGELOG.md]: Updated the docs now.

**Comment by Leo-XM-Zeng** [comment]
Hi everyone,

Does PG14 need to be added to the current Docker? 
It has not been added yet and needs to be modified if necessary.

**Review by Y-- — APPROVED** [review]

**Pull request merged**